File: Java-Codes/1.java
//Gaurav Sharma

class sumcommand {
    public static void main(String args[]) {
        int l = Integer.parseInt(args[0]);
        int sum = 0;
        for (int i = 1; i <= l; i++) {
            sum += (Integer.parseInt(args[i]));
        }
        System.out.print("sum is :" + sum);
        System.out.print('\n');
    }

}

File: Cpp-Codes/1.cpp
//1. WAP to print the sum and product of digits of an integer.
//Gaurav Sharma

#include<iostream>
#include<conio.h>
using namespace std;
int main()
{int a,s=0,p=1;
cout<<"enter the integer:";
cin>>a;
do
{int x;
x=a%10;
s=s+x;
p=p*x;
a=a/10;
}while(a!=0);
cout<<endl<<"sum of the digits :"<<s;
cout<<endl<<"product of the digits:"<<p;
getch();
return 0;
}
